client/additional-methods-setup/methods-selector/add-payment-methods-task.js

```diff
@@ -113,22 +171,18 @@ const AddPaymentMethodsTask = () => {
 							) }
 						</p>
 						<PaymentMethodCheckboxes>
-							<PaymentMethodCheckbox
```

Comment:
We don't really have a "Popular with customers in {country name}" ability to determine payment methods, so I did my best to group them - CC + all the others.